packages/toolkit/src/view/dashboard/DashboardPipelinesTable.tsx

```diff
@@ -19,8 +19,8 @@ export const DashboardPipelinesTable = (
   props: DashboardPipelinesTableProps
 ) => {
   const router = useRouter();
-  const { entity } = router.query;
-  const { days } = router.query;
+  const { entity } = useParams();
+  const { days } = useParams();
```

Comment:
Please be careful of duplication

```suggestion
  const { entity, days } = useParams();
```